drivers/spi/spi_ll_stm32.c

```diff
@@ -678,6 +709,52 @@ static int spi_stm32_release(const struct device *dev,
 	return 0;
 }
 
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi)
+static int32_t spi_stm32_count_bufset_frames(const struct spi_config *config,
+			const struct spi_buf_set *bufs)
```

Comment:
Nitpick: indentation

Reply:
Done (I hope)

Reply:
> A very commonly used style is to align descendants to a function open parenthesis.

I have tried this and the compliance check fails. 

Compliance checks pass now, so marking as resolved.